WPFHospitalEditor/ScheduleWindow.xaml

```diff
@@ -0,0 +1,35 @@
+﻿<Window x:Class="WPFHospitalEditor.ScheduleWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:local="clr-namespace:WPFHospitalEditor"
+        mc:Ignorable="d"
+        Title="Schedule Appointment" Height="200" Width="350" ResizeMode="NoResize" WindowStyle="None" WindowStartupLocation="CenterScreen">
+    <Window.Resources>
+        <Style TargetType="{x:Type Button}">
+            <Setter Property="VerticalAlignment" Value="Center"/>
+            <Setter Property="Width" Value="100"/>
+            <Setter Property="Height" Value="25"/>
+            <Setter Property="Background" Value="SkyBlue"/>
+            <Setter Property="Foreground" Value="White"/>
+            <Setter Property="BorderThickness" Value="0"/>
+        </Style>
+    </Window.Resources>
+    <Grid>
+        <Border BorderThickness="10" BorderBrush="#FF8BD9E0" x:Name="Border">
+            <Grid x:Name="DynamicGrid" Background="#FFC6F5F8">
+                <StackPanel Orientation="Horizontal" HorizontalAlignment="Center" VerticalAlignment="Top" Margin="0,10,0,0">
+                    <ComboBox Name="patients" Width="200"/>
```

Comment:
Using a combo box to find a patient is not a practical way!

Reply:
Will be fixed in issue :)